components/multi-platform-controller/production-downstream/kflux-ocp-p01/host-config.yaml

```
@@ -522,7 +539,22 @@ data:
   dynamic.linux-large-ppc64le.max-instances: "10"
   dynamic.linux-large-ppc64le.allocation-timeout: "1800"
 
-  # Same as linux-large-ppc64le but with 200GB disk instead of default 100GB
+  #PPC64LE CPU dynamic nodes
+  dynamic.linux-largecpu-ppc64le.type: ibmp
+  dynamic.linux-largecpu-ppc64le.ssh-secret: "internal-prod-ibm-ssh-key"
+  dynamic.linux-largecpu-ppc64le.secret: "internal-prod-ibm-api-key"
+  dynamic.linux-largecpu-ppc64le.key: "prod-konflux-infra"
+  dynamic.linux-largecpu-ppc64le.image: "ppc64le-rhel-94-base-27nov2024"
+  dynamic.linux-largecpu-ppc64le.crn: "crn:v1:bluemix:public:power-iaas:dal12:a/eebc7153d0c844f9915fb6f3b9850a3b:d6faf373-c0b6-4ba9-b0bc-4f7e0a74f9d5::"
+  dynamic.linux-largecpu-ppc64le.url: "https://us-south.power-iaas.cloud.ibm.com"
+  dynamic.linux-largecpu-ppc64le.network: "4d164943-b043-4c6e-bba5-d233abf66c6a"
+  dynamic.linux-largecpu-ppc64le.system: "e980"
+  dynamic.linux-largecpu-ppc64le.cores: "8"
+  dynamic.linux-largecpu-ppc64le.memory: "16"
+  dynamic.linux-largecpu-ppc64le.max-instances: "5"
+  dynamic.linux-largecpu-ppc64le.allocation-timeout: "1800"
+
+# Same as linux-large-ppc64le but with 200GB disk instead of default 100GB
```

nit: you removed the indentation, let's put it back
```suggestion
  # Same as linux-large-ppc64le but with 200GB disk instead of default 100GB
```